security/mkcert/Portfile

```diff
@@ -0,0 +1,55 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+
+github.setup        FiloSottile mkcert 1.1.0 v
+
+platforms           darwin
+categories          security devel
+license             permissive
```

Comment:
I have the same question, I'm not sure this is a valid license name. I'll invoke someone who would know.

Reply:
The list of good licenses is here <https://trac.macports.org/browser/trunk/base/portmgr/jobs/port_binary_distributable.tcl>, and `permissive` is one of them.

Reply:
@kencu We should link to that from the guide or some such. It's hard to find and it comes up a lot.

Reply:
Agreed, it was difficult to find the list of licenses, and I only stumbled onto the Trac wiki entry after a healthy amount of Googling.

Reply:
Make this "BSD", since this is the BSD-3-Clause license.